src/servo/joint_controller.cpp

```diff
@@ -65,10 +65,26 @@ void JointController::legs_state_update_callback(const hexapod_msgs::LegsJoints:
     servo_controller->set_angle(RB_TIBIA_CH, preprocess(legs_joints->right_back_leg.tibia, RB_TIBIA_DIR, RB_TIBIA_OFFSET));
 }
 
+void JointController::head_state_update_callback(const hexapod_msgs::HeadJoints::ConstPrt &head_joins) {
+    auto preprocess = [](float angle_deg, bool rotation_dir, float offset_deg){
```

Comment:
Can you move the `preprocess` lambda function to a private function so we don't have the same function in both callback functions?